WordPress/src/main/java/org/wordpress/android/resolver/DbWrapper.kt

```diff
@@ -6,5 +6,6 @@ import javax.inject.Inject
 
 @Reusable
 class DbWrapper @Inject constructor() {
-    fun giveMeWritableDb() = WellSql.giveMeWritableDb()
+    fun giveMeWritableDb() = WellSql.giveMeWritableDb()!!
```

Comment:
This is not a request for change but a note to revisit if we can avoid force unwrapping here 👍